theia-example-updater/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "private": true,
+  "name": "theia-example-updater",
+  "version": "1.7.0",
+  "description": "Theia - Updater and branding for the installer",
+  "dependencies": {
+    "@theia/core": "^1.7.0",
+    "@theia/output": "^1.7.0",
+    "electron-updater": "^4.3.5",
+    "electron-log": "^4.3.0"
+  },
+  "devDependencies": {
+    "rimraf": "^2.7.1",
+    "typescript": "^3.9.2",
+    "tslint": "^5.12.0"
+  },
+  "theiaExtensions": [
+    {
+      "electronMain": "lib/electron-main/update/theia-updater-main-module",
+      "frontendElectron": "lib/electron-browser/theia-updater-frontend-module"
+    }
+  ],
+  "keywords": [
+    "theia-extension"
+  ],
+  "license": "EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/eclipse-theia/theia-example.git"
+  },
+  "bugs": {
+    "url": "https://github.com/eclipse-theia/theia-example/issues"
+  },
+  "homepage": "https://github.com/eclipse-theia/theia-example",
+  "files": [
+    "lib",
+    "src"
+  ],
+  "scripts": {
+    "prepare": "yarn run clean && yarn run build",
+    "clean": "rimraf lib",
+    "build": "tsc --build base.tsconfig.json"
```

Comment:
This should use a local `tsconfig.json` file which inherits from the base one. The base one should be put in the root of the monorepo for use/exension by other packages